### PR TITLE
Post accessibility notification in presenter

### LINKED ISSUE #60: Add accessibility notification support for in-app notifications
These in-app notifications don't come with UIAcessibility announcements/notifications, making them quite very easy to miss for VoiceOver users; make sure we post a notification for them, like ±so:

```swift
UIAccessibility.post(notification: .announcement, argument: notification.title)
```

when showing the notification.

### PR DESCRIPTION
**Pull request by BasThomas**
Fixes #60

I tried adding a test that checks the `UIAccessibility.announcementDidFinish` notification, but that somehow does not arrive in tests. :/

### CODE REVIEW

**Comment by BasThomas** [comment]
Looks like CI failed because it tried running `fastlane` from Bundler, but I don't think we host it there. @AvdLee?

**Review by kairadiagne — APPROVED** [review]

File: Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift [hunk 1]
```diff
@@ -35,6 +35,14 @@ public final class UINotificationEaseOutEaseInPresenter: UINotificationPresenter
             presentationContext.notificationView.topConstraint?.constant = presentationContext.notificationView.layoutMargins.top + UIApplication.statusBarHeight
         }
 
+        let content = presentationContext.notification.content
+        let accessibilityAnnouncement = [content.title, content.subtitle]
+            .compactMap { $0 }
+            .filter { $0 != "" }
```
Comment (amirdew) [inline Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift:41]:
```suggestion
            .filter { !$0.isEmpty }
```

**Review by amirdew — APPROVED** [review]

**Comment by BasThomas** [comment]
> error: the library 'DangerDependencies' requires macos 10.10, but depends on the product 'WeTransferPRLinter' which requires macos 10.15; consider changing the library 'DangerDependencies' to require macos 10.15 or later, or the product 'WeTransferPRLinter' to require macos 10.10 or earlier.

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Testing cancelled because the build failed.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Command PhaseScriptExecution failed with a nonzero exit code</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L15** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L19** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L22** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L27** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L31** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L50** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L57** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L61** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **Sources/UINotificationPresenters/UINotificationEaseInOutPresenter.swift#L63** - Lines should not have trailing whitespace. (`trailing_whitespace`)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/fdb98d17-064d-4580-ac0e-d258025babc4" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against cc0f5dfe03f6f99c6b85cb07c3839d54011ce8f8
</p>

**Comment by BasThomas** [comment]
Seems like SwiftLint fails to run properly?

**Review by AvdLee — APPROVED** [review]

**Pull request merged**